src/lwc/rd2EntryForm/rd2EntryForm.html

```diff
@@ -141,9 +151,15 @@ <h3 class="slds-section__title slds-theme_shade">
                                             is-edit-mode={isEdit}
                                             card-exp-date={cardExpDate}
                                             card-last-four-label={cardLastFourLabel}
+                                            ach-last-four-label={achLastFourLabel}
                                             card-last-four={cardLastFour}
+                                            ach-last-four={achLastFour}
                                             is-edit-payment={isEdit}
```

Comment:
We have isEdit bound twice here (151 and 157) - I'd like to try to de-duplicate it but when I did it broke the rd2EditPaymentInformationModal component, nothing rendered.

Reply:
The isEditMode is the toggle between editing and displaying the edit information. isEditPayment is the option to opt out and not use Elevate. I think we can have a better naming for isEditPayment to make it less confusing